File: examples/UCSB-CHG/UCSB-CHG_CHIRPS_PENTAD.js
var dataset = ee.ImageCollection('UCSB-CHG/CHIRPS/PENTAD')
                  .filter(ee.Filter.date('2018-05-01', '2018-05-05'));
var precipitation = dataset.select('precipitation');
var precipitationVis = {
  min: 0.0,
  max: 112.0,
  palette: ['001137', '0aab1e', 'e7eb05', 'ff4a2d', 'e90000'],
};
Map.setCenter(17.93, 7.71, 2);
Map.addLayer(precipitation, precipitationVis, 'Precipitation');

File: examples/UCSB-CHG/UCSB-CHG_CHIRPS_PENTAD.py
dataset = ee.ImageCollection('UCSB-CHG/CHIRPS/PENTAD').filter(
    ee.Filter.date('2018-05-01', '2018-05-05')
)

precipitation = dataset.select('precipitation')

precipitation_vis = {
    'min': 0,
    'max': 112,
    'palette': ['001137', '0aab1e', 'e7eb05', 'ff4a2d', 'e90000'],
}

m = geemap.Map()
m.set_center(17.93, 7.71, 2)
m.add_layer(precipitation, precipitation_vis, 'Precipitation')
m
